test/nbrowser/DocTour.ts

```diff
@@ -47,51 +47,175 @@ describe('DocTour', function () {
     const docTour = await driver.executeScript('return window._gristDocTour()');
     assert.deepEqual(docTour, [
       {
-        body: 'General Kenobi!',
+        body: '<span>' +
+          '<p>General Kenobi!</p>\n' +
+          '</span>',
         placement: 'bottom',
         selector: '.active_cursor',
         showHasModal: false,
         title: 'Hello there!',
         urlState: {colRef: 2, rowId: 2, sectionId: 1}
       },
       {
-        body: 'no title',
+        body: '<span>' +
+          '<p>no title</p>\n' +
+          '</span>',
         placement: 'auto',
         selector: '.active_cursor',
         showHasModal: true,
         title: '',
         urlState: null,
       },
       {
-        body: '<div>' +
-          '<p></p>' +
+        body: '<span>' +
+          '<span></span>' +
           '<p><div class="_grainXXX_">' +
           '<a href="https://www.getgrist.com/" target="_blank" class="_grainXXX_ _grainXXX_">' +
           '<div class="_grainXXX_ _grainXXX_" style="mask-image: var(--icon-Page);"></div>' +
           'A link with an icon' +
           '</a>' +
           '</div></p>' +
-          '</div>',
+          '</span>',
         placement: 'auto',
         selector: '.active_cursor',
         showHasModal: false,
         title: 'no body',
         urlState: {colRef: 4, rowId: 4, sectionId: 1}
       },
       {
-        body: '<div>' +
-          '<p>Good riddance</p>' +
+        body: '<span><span>' +
+          '<p>Good riddance</p>\n</span>' +
           '<p><div class="_grainXXX_">' +
           '<a href="https://xkcd.com/" target="_blank" class="_grainXXX_ _grainXXX_">' +
           'Test link here' +
           '</a>' +
           '</div></p>' +
-          '</div>',
+          '</span>',
         placement: 'auto',
         selector: '.active_cursor',
         showHasModal: true,
         title: 'Bye',
         urlState: null,
+      },
+      {
+        body: '<span><p>' +
+          '<strong>bold</strong>' +
```

Comment:
Nice! I think there may be a reasonably simple way to include a test of plaintext without a new fixture by switching the column type using API from the test (using `PATCH /api/docs/{docId}/tables/GristDocTour/columns` endpoint) in a test case and running some more checks. But also not so important and OK to omit.

The important thing is to test manually that both plaintext and markdown look as expected (e.g. that this doesn't change behavior of plaintext in some unintended way).